examples/Receiver_Response.py

```diff
@@ -3,18 +3,22 @@
 
 Two files are required for the following example:
 
-    File1 contains the analytic signal acquired on the GDS oscilloscope
-    directly output from the AFG output. Each node pertains to signal with a
-    different frequency (in kHz) which are fit to a complex function to extract
-    the :math:`V_{p}` (in μV).
-    File2 contains the quadrature signal acquired on the receiver when the same
+file1
+    contains the analytic signal acquired on the (GW Instek GDS) oscilloscope
+    directly captured from the rf source (GW Instek AFG). Each node pertains to
+    signal with a different frequency (in kHz) which are fit to a complex
+    function to extract the :math:`V_{p}` (in μV).
+
+file2
+    contains the quadrature signal acquired on the receiver when the same
```

Comment:
since we are laying this out like a list (the ":") make each file arranged like we would the list of parameters

Reply:
this is an RST formatting issue